examples/with-federated-authentication/README.md

```diff
@@ -0,0 +1,49 @@
+# Example: `with-federated-authentication`
+
+This example shows how to create a sub organization and faciliate creating a private key for the user's sub organization with Turnkey.
+
+## Getting started
+
+### 1/ Cloning the example
+
+Make sure you have `Node.js` installed locally; we recommend using Node v16+.
+
+```bash
+$ git clone https://github.com/tkhq/sdk
+$ cd sdk/
+$ corepack enable  # Install `pnpm`
+$ pnpm install -r  # Install dependencies
+$ pnpm run build-all  # Compile source code
+$ cd examples/with-federated-authentication/
+```
+
+### 2/ Setting up Turnkey
+
+The first step is to set up your Turnkey organization and account. By following the [Quickstart](https://turnkey.readme.io/docs/quickstart) guide, you should have:
+
+- A public/private API key pair for Turnkey
+- An organization ID
+- A (crypto) private key ID
+
+Once you've gathered these values, add them to a new `.env.local` file. Notice that your private key should be securely managed and **_never_** be committed to git.
```

Comment:
Just to make sure there's no ambiguity: "Notice that your API private key should be...etc..." (to minimize confusion with crypto private keys)